.github/workflows/core_windows_build.yml

```diff
@@ -26,9 +26,8 @@ jobs:
 
       - name: Checkout Submodules
         shell: bash
-        run: |
-          git submodule init && git submodule update
-  
+        run: apps/ci/ci-submodule-update.sh
```

Comment:
This line should be reverted back to how it was, since Shell files can not run in Windows.